test/server/lib/route-helpers.js

```diff
@@ -0,0 +1,124 @@
+'use strict';
+
+const moment = require('moment');
+const timeshift = require('timeshift');
+const stub = sinon.stub().resolves('finished');
+const { dateChecker, promiseResponder, dashboard, home } = require('proxyquire')('../../../src/lib/route-helpers', {
+	'./model': stub
+});
+
+describe('lib/routesHelpers', () => {
+	describe('dateChecker', () => {
+			describe('checks for invalid dates', () => {
+				it('should return false when the input is not a valid date', () =>
+					expect(dateChecker('2019-09-50'))
+						.to.equal('Invalid date'));
+			});
+			describe('checks for valid dates', () => {
+				it('should return an ISO formatted timestamp string', ()=> {
+					expect(dateChecker('2019-06-03'))
+						.to.be.a('string')
+						.and.to.equal('2019-06-03T00:00:00+01:00');
+					expect(dateChecker('2019-12-03'))
+						.to.be.a('string')
+						.and.to.equal('2019-12-03T00:00:00Z');
```

Comment:
FWIW, this would be a nice place to use a 'quickcheck' approach. This is because the operation is reversible meaning that we could test that we can regenerate the input. (And then test it with hundreds or thousands of random dates.)

That's not something for this PR of course but it might be a nice exercise for an apprentice to attempt.

Reply:
Haha, yeh agreed, **tag** you're it @aarongill1 !